html5lib/constants.py

```diff
@@ -2939,8 +2939,14 @@
 
 
 class DataLossWarning(UserWarning):
+    """Raised whenever html5lib changes something in such a way that some input data is lost"""
     pass
 
 
 class ReparseException(Exception):
+    """Raised when the parser should restart parsing
```

Comment:
This should really be internal; it should never be raised through any public interface.

Reply:
The way the docs are structured, it's difficult to make this not show up in the docs. I'll take away the docstring, but that's how it's going to have to live for 1.0.

Reply:
We should probably just `_` prefix it, I guess?

Reply:
Sure! If that's important, someone who is not me can write up an issue for making those changes and make them. If that makes it in time for 1.0, that's cool. Otherwise I'm not worrying about this.